scene/main/viewport.cpp

```diff
@@ -3222,7 +3223,12 @@ Viewport::Viewport() {
 	gui.tooltip_timer = -1;
 
 	//gui.tooltip_timer->force_parent_owned();
-	gui.tooltip_delay = GLOBAL_DEF("gui/timers/tooltip_delay_sec", 0.5);
+	if (isEngine()) {
```

Comment:
This method doesn't exist...

Reply:
Yeah, this is a placeholder for a test if the code is run in the editor. How can I do that?

Reply:
`Engine::get_singleton()->is_editor_hint()`

Reply:
Thanks, I will change that!

Reply:
That code would be problematic though, as when running in the editor you would no longer define `gui/timers/tooltip_delay_sec`, so it won't appear in the Project Settings to set for the running game. And the call to `set_custom_property_info` afterwards would fail/run on an undefined value.

So you should keep this `GLOBAL_DEF` even when running in the editor, but you can then define the `gui.tooltip_delay` to the EditorSettings parameter when `Engine::get_singleton()->is_editor_hint()`.

Reply:
Yes, that sounds like a far cleaner approach.

Reply:
When I have
```cpp
gui.tooltip_delay = EditorSettings::get_singleton()->get("interface/editor/code_font_size");
```
how can I make it not crash in the project selection?